yarn-project/end-to-end/src/shared/cli.ts

```diff
@@ -121,7 +121,7 @@ export const cliTestSuite = (
       const ownerAddress = AztecAddress.fromString(foundAddress!);
 
       debug('Deploy Token Contract using created account.');
-      await run(`deploy TokenContractArtifact --salt 0 --args ${ownerAddress}`);
+      await run(`deploy TokenContractArtifact --salt 0 --args ${ownerAddress} 'TokenName' 'TKN' 18`);
```

Comment:
probably also have to update in the docs

Reply:
The docs should be reading from the values in the test, so it is already there. 
![image](https://github.com/AztecProtocol/aztec-packages/assets/16536249/c5d04aed-1149-4b9c-8345-6a879507bfda)